build.sbt

```diff
@@ -34,6 +36,8 @@ lazy val igluCore = (project in file("."))
 lazy val igluCoreJson4s = (project in file("iglu-core-json4s"))
   .dependsOn(igluCore)
   .settings(json4sBuildSettings: _*)
+  .settings(BuildSettings.mimaSettings)
```

Comment:
Wouldn't it be implied since we have it enabled in `igluCore`?

Reply:
Nope, turns out that it doesn't carry over. I think the idea is that you may want different settings per project.